CHANGES.md

```diff
@@ -1,5 +1,16 @@
 # Changes
 
+## edge-22.1.2
+
+This release sets the version of the extension Helm charts to 30.0.0-edge to
+ensure that previous versions of these charts can be upgraded properly.
+
+* Reset extensions Helm chart versions at 30.0.0-edge
+* Pin multicluster extension pause container version to 3.2 so that it will work
+  on m1 macs
```

Comment:
This really impacted all Arm architectures
```suggestion
* Pin multicluster extension pause container version to 3.2 so that it will work
  on Arm architectures
```